client/js/components/support/DpSupport.vue

```diff
@@ -24,12 +24,12 @@ All rights reserved
     </h3>
     <ul
       class="u-mb-0_75"
-      :class="contactCount === 1 ? '' : 'grid lg:grid-cols-3 gap-3'">
+      :class="{ 'grid lg:grid-cols-3 gap-3': visibleContacts.length !== 1 }">
```

Comment:
How does this work? The classes are only added if the condition is true?

Reply:
yes, right